test/journeys/lib/test-helpers/recents-widget/index.js

```diff
@@ -28,15 +28,9 @@ export function displayIncomingMessage(aBrowser, sender, conversation, message,
   waitForPromise(sender.spark.internal.conversation.post(conversation, {
     displayName: message
   }));
-  aBrowser.waitUntil(() =>
-    aBrowser.element(`${elements.firstSpace} ${elements.title}`).getText() === spaceTitle
-    , 5000
-    , 'conversation not displayed');
-  aBrowser.waitUntil(() =>
-    aBrowser.element(`${elements.firstSpace} ${elements.lastActivity}`).getText().includes(message)
-    , 5000
-    , 'does not have last message displayed');
-  assert.isTrue(aBrowser.element(`${elements.firstSpace} ${elements.unreadIndicator}`).isVisible(), 'does not have unread indicator');
+  aBrowser.waitUntil(() => aBrowser.getText(`${elements.firstSpace} ${elements.title}`) === spaceTitle);
```

Comment:
The spaced out version of `waitUntil` was so that we could have proper error messages on failures. Was this a cause for a problem?

Reply:
I thought it never actually printed the provided error message with waitUntil because of our multiremote woes? Or am I remembering incorrectly? I've never actually seen one of the custom error messages get printed...

Reply:
It does print the error messages properly, but if it is stabilizing tests due to random timeout values, then feel free to remove them.